.github/workflows/ci.yaml

```diff
@@ -18,6 +18,9 @@ jobs:
                     -   php: '8.2'
                         deps: lowest
                         deprecations: max[self]=0
+                    -   php: '8.4'
+                        symfony: '7.4.*'
+                        deps: highest
```

Comment:
Including only the Symfony LTS version explicitly, as recommended by @stof in https://github.com/symfony/monolog-bundle/pull/518#discussion_r2376432602